api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/TestConstants.java

```diff
@@ -31,4 +31,7 @@
     public static final LocalDate OVERRIDE_UNTIL_FIVE_YEARS = LocalDate.now().plusYears(5);
     public static final List<LocalDate> CHILD_BORN_IN_FUTURE = singletonList(LocalDate.now().plusDays(1));
 
+    //NINOs
+    public static final String NINO = "MC125456C";
```

Comment:
You do not need to do this, there are loads of test NINOs in the common-java repository, the idea was not to litter this file with them too, so lets remove this and use one of the standard ones (HOMER_NINO or MARGE_NINO for example).

Reply:
We need a third NINO for one of the tests, I don't think we have a third currently?

Reply:
I'll add this one to common-java so we can remove it from here, leave that to me.

Reply:
You do need to be careful with this NINO, it's the same as the HOME_NINO value so it may cause issues and failing test, so I'd suggest changing it.

Reply:
Actually, I've just spotted it is very slightly different so its ok. I'll make it substantially different in the one in common-java so its more obvious.